tests/bsim/bluetooth/host/iso/cis/src/cis_central.c

```diff
@@ -234,10 +239,24 @@ static void reconfigure_cig(void)
 		return;
 	}
 
+	/* Test modifying to different values for both intervals */
+	param.num_cis = 0U;
+	param.c_to_p_interval = 5000U; /* us */
+	param.p_to_c_interval = 2500U; /* us */
+
+	err = bt_iso_cig_reconfigure(cig, &param);
+	if (err != 0) {
+		FAIL("Failed to reconfigure CIG to different intervals (%d)\n", err);
+
+		return;
+	}
```

Comment:
Should we have something similar for different values of latency?

Reply:
yes, I don't think we are checking changing latencies at the moment but I can add it

Reply:
Come to think about it, arguably we should do the same for the other parameters in CIG create

Reply:
> Come to think about it, arguably we should do the same for the other parameters in CIG create

Yeah, we probably want to improve ISO host test coverage :) 

The GMAP and CAP tests are doing quite a few different things, but doesn't hurt to have tests running at the ISO layer to also verify things like that

Reply:
for now just added the test for reconfiguring latency